src/client/App.js

```diff
@@ -1,11 +1,12 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import Header from './Header';
 import Search from './search/Search';
 import Feed from './feed/Feed'
 import MostUsedWords from './results/MostUsedWords';
 import { groupedCountWords, mostFrequentWords, mostActiveUsers, exampleTweets } from './tweetAnalysis';
 
-const App = () => {
+const App = ({state}) => {
   return (
```

Comment:
Can we remove the return keyword from here and the mapStateToProps

Also this is tabbed not spaced